project_config.yml

```
@@ -0,0 +1,24 @@
+  - arrival_date
+  - no_of_previous_cancellations
+  - no_of_previous_bookings_not_canceled
+  - avg_price_per_room
+  - no_of_special_requests
+cat_features:
+  - type_of_meal_plan
+  - required_car_parking_space
+  - room_type_reserved
+  - market_segment_type
+  - repeated_guest
+target: booking_status
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)